package.json

```diff
@@ -1,39 +1,53 @@
 {
-  "name": "tsc-sample",
+  "name": "designity",
   "version": "1.0.0",
-  "description": "",
+  "description": "Designity Project",
   "main": "index.js",
-  "type": "commonjs",
   "scripts": {
     "build": "npx tsc",
-    "start": "node dist/index.js",
-    "dev": "npx tsc -w & nodemon dist/index.js"
+    "dev": "nodemon ./src/index.ts"
```

Comment:
we need a script to run Temporal worker with it

Reply:
Done